depthai_examples/ros1_src/mobilenet_publisher.cpp

```diff
@@ -37,77 +37,70 @@ dai::Pipeline createPipeline(bool syncNN, std::string nnPath){
+    if(pnh.hasParam("nn_path")) {
+        pnh.getParam("nn_path", nnPath);
     }
 
     dai::Pipeline pipeline = createPipeline(syncNN, nnPath);
     dai::Device device(pipeline);
-    
+
     std::shared_ptr<dai::DataOutputQueue> previewQueue = device.getOutputQueue("preview", 30, false);
     std::shared_ptr<dai::DataOutputQueue> nNetDataQueue = device.getOutputQueue("detections", 30, false);
 
     std::string color_uri = cameraParamUri + "/" + "color.yaml";
 
     dai::rosBridge::ImageConverter rgbConverter(tfPrefix + "_rgb_camera_optical_frame", false);
     dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame> rgbPublish(previewQueue,
-                                                                                     pnh, 
-                                                                                     std::string("color/image"),
-                                                                                     std::bind(&dai::rosBridge::ImageConverter::toRosMsg, 
-                                                                                     &rgbConverter, // since the converter has the same frame name
-                                                                                                      // and image type is also same we can reuse it
-                                                                                     std::placeholders::_1, 
-                                                                                     std::placeholders::_2) , 
-                                                                                     30,
-                                                                                     color_uri,
-                                                                                     "color");
-
+                                                                                  pnh,
+                                                                                  std::string("color/image"),
+                                                                                  std::bind(&dai::rosBridge::ImageConverter::toRosMsg,
+                                                                                            &rgbConverter,  // since the converter has the same frame name
+                                                                                                            // and image type is also same we can reuse it
+                                                                                            std::placeholders::_1,
+                                                                                            std::placeholders::_2),
+                                                                                  30,
+                                                                                  color_uri,
+                                                                                  "color");
```

Comment:
this seems to have messed up in the changes form the formatting. Can we try and adjust it to fix this ?

Reply:
Not here. may be in a separate PR. Applies to all the files that has this part shifted like that.